calico_containers/pycalico/datastore.py

```diff
@@ -745,6 +745,43 @@ def update_endpoint(self, endpoint):
         endpoint._original_json = new_json
 
     @handle_errors
+    def create_endpoint(self, hostname, orchestrator_id, workload_id,
```

Comment:
You'll want to mention in the docstring that this does not store the endpoint in ETCD, as that should be done with endpoint_set or endpoint_update, as appropriate.